scs-app/src/app/services/credit-calc/credit-calc.service.ts

```diff
@@ -0,0 +1,17 @@
+export class CreditCalcService {
+
+  private courselist: {course_name: string, fulfills: string}[] = [
+    {'course_name': 'CSC 212 - Programming with Data Structures', fulfills: 'core_data'},
```

Comment:
camelNotation :) 

perhaps we should store this info on the db as well

Reply:
yeah the full course list could totally live in the db if that's better